packages/rocket-uploads3-store-event-aws-infrastructure/src/lambdas/index.ts

```diff
@@ -0,0 +1,44 @@
+import { AWSError, DynamoDB } from 'aws-sdk'
+import { PutItemOutput } from 'aws-sdk/clients/dynamodb'
+
+export const handler = async (event: any): Promise<any> => {
+  const { v4: uuidv4 } = require('uuid')
+  const ddb = new DynamoDB.DocumentClient()
+
+  const {
+    bucket: { name },
+    object: { key, size },
+  } = event.Records[0].s3
+  const fileSize = size
+  const s3uri = `s3://${name}/${key}`
+
+  const boosterEvent = {
+    s3uri: s3uri,
+    fileSize: fileSize,
+  }
```

Comment:
Maybe this can be a function like this?

```suggestion
const boosterEvent = s3EventToBoosterEvent(event)

// ...
// somewhere else in the module
function s3EventToBoosterEvent(event: /* type of the event */): {s3uri: string, fileSize: string} {
  const {
    bucket: { name },
    object: { key, size },
  } = event.Records[0].s3
  const fileSize = size
  const s3uri = `s3://${name}/${key}`

  return {
    s3uri,        // you can omit setting the value if the variable for the value has the same name as the key
    fileSize,
  }
}
```

Reply:
yep makes sense